### PR TITLE
Dump out package management packages

### PR DESCRIPTION
**Pull request by praiskup**

### CODE REVIEW

**Comment by praiskup** [comment]
Output looks like: ```
INFO: Buildroot is handled by package management installed into bootstrap:
rpm-4.14.3-26.el8.x86_64
python3-dnf-4.7.0-16.el8_8.alma.noarch
python3-dnf-plugins-core-4.0.21-19.el8_8.noarch
package yum is not installed
package yum-utils is not installed
```

**Comment by praiskup** [comment]
The output now looks like (the `--no-bootstrap-chroot` variant):
```
INFO: Buildroot is handled by package management available on this host:
  rpm-4.18.1-3.fc38.x86_64
  rpm-sequoia-1.5.0-1.fc38.x86_64
  python3-dnf-4.16.2-1.fc38.noarch
  python3-dnf-plugins-core-4.4.2-1.fc38.noarch
  yum-4.16.2-1.fc38.noarch
```

File: mock/py/mockbuild/package_manager.py [hunk 1]
```diff
@@ -166,6 +166,46 @@ def adjust_command_options(self, command, opts):
...
+                _do("downloaded with a bootstrap image", cmd,
+                    chrootPath=self.bootstrap_buildroot.make_chroot_path())
+            else:
+                # rpm installed into bootstrap by host's package management
+                _do("installed into bootstrap", cmd + [
+                    "--root", self.bootstrap_buildroot.make_chroot_path()
+                ])
+        else:
+            # Execute with installroot from host
+            _do("available on this host", cmd)
```
Comment (xsuchy) [inline mock/py/mockbuild/package_manager.py:207]: I would suggest wording "from host and used with --installroot"

**Comment by xsuchy** [comment]
Otherwise LGTM

**Comment by praiskup** [comment]
Updated, and added a release-notes entry.

**Review by xsuchy — APPROVED** [review]
+1

**Pull request merged**